csrc/transformer/inference/csrc/layer_norm.cu

```diff
@@ -200,7 +200,11 @@ Template arg:
     StoreResidual: controls whether the residual calculation is stored
         or not. When set to false, the input `res_output` is unused.
 */
-template <typename T, int UNROLL, bool StoreResidual>
```

Comment:
For non `typename T` template parameters, let's use `camelCase` moving forward to consistently differentiate the variables from template params.

Reply:
done